nttt/utilities.py

```diff
@@ -53,4 +54,29 @@ def find_files(src, file_names=[], extensions=[]):
             if valid_file:
                 files_found.append(fpath)
 
-    return files_found
\ No newline at end of file
+    return files_found
+
+
+#PUB-9 utilities
```

Comment:
Unless the name of a function and its arguments makes clear what the function does, it is wise to add some documentation above a function, containing the purpose of the function (what it does, not how it does it), a description of its arguments and a description of the return value, if present.
For example, this function name contains the word special. What kind of special thing does this function do?